packages/go/analysis/azure/post_test.go

```diff
@@ -42,14 +43,21 @@ var (
 // setupRoleAssignments is used to create a testable RoleAssignments struct. It is used in all RoleAssignments tests
 // and may require adjusting tests if modified
 func setupRoleAssignments() azure.RoleAssignments {
+	roleMap := map[string]*roaring.Bitmap{
+		constants.GlobalAdministratorRoleID:   roaring.New(),
+		constants.ReportsReaderRoleID:         roaring.New(),
+		constants.HelpdeskAdministratorRoleID: roaring.New(),
+		constants.PartnerTier1SupportRoleID:   roaring.New(),
+	}
+	roleMap[constants.GlobalAdministratorRoleID].Add(user.ID.Uint32())
+	roleMap[constants.ReportsReaderRoleID].Add(group.ID.Uint32())
+	roleMap[constants.HelpdeskAdministratorRoleID].Add(group.ID.Uint32())
+	roleMap[constants.PartnerTier1SupportRoleID].Add(app.ID.Uint32())
```

Comment:
In this same file, we're using `uint32(user.ID)` and this method. Honestly, not a huge fan that we made a method that isn't any shorter than just doing an explicit type conversion, it feels like it's an unnecessary indirection, but we should probably at least be consistent in how we approach it per file